controls/SV-204455.rb

```diff
@@ -7,24 +7,32 @@
     unintentional reboot from the Ctrl-Alt-Delete sequence is reduced because the user will be prompted before any
     action is taken.'
   desc 'check', 'Verify the operating system is not configured to reboot the system when Ctrl-Alt-Delete is pressed.
-    Check that the ctrl-alt-del.target is masked and not active with the following command:
-    # systemctl status ctrl-alt-del.target
-    ctrl-alt-del.target
-    Loaded: masked (/dev/null; bad)
-    Active: inactive (dead)
-    If the ctrl-alt-del.target is not masked, this is a finding.
-    If the ctrl-alt-del.target is active, this is a finding.'
-  desc 'fix', 'Configure the system to disable the Ctrl-Alt-Delete sequence for the command line with the following
-    command:
-    # systemctl mask ctrl-alt-del.target'
+
+Check that the ctrl-alt-del.target is masked and not active with the following command:
+
+$ sudo systemctl status ctrl-alt-del.target
+
+ctrl-alt-del.target
+Loaded: masked (/dev/null; bad)
```

Comment:
The InSpec code does not check for the value to be masked. When testing locally, the InSpec test passed when the the system's response to the command was:
```
Loaded: loaded (/usr/lib/systemd/system/reboot.target: disabled: vendor present: disabled)
Actice: inactive (dead)
```

and when the response was:
```
Loaded: masked (/dev/null: bad)
Active: inactive (dead)
```

Reply:
Proposed code:
```rb
service_status = systemd_service('ctrl-alt-del.target').params

describe service_status do
  its('LoadState') { should cmp 'masked' }
  its('ActiveState') { should cmp 'inactive' }
end
```

Need to run it by Aaron to understand the HashieMash output

Reply:
Split into two describe blocks to avoid the HashieMash output and keep it simple